libvast/src/option_declaration_set.cpp

```diff
@@ -124,32 +124,38 @@ std::string option_declaration_set::usage() const {
     auto& shorts = x.short_names();
     arg << "  "; 
     if (!shorts.empty()) {
-      arg << "-" <<shorts[0];
-      arg << " [";
+      auto i = shorts.begin();
+      auto e = shorts.end();
+      arg << "-" << *i << " [";
+      for (++i; i != e; ++i)
+        arg << "-" << *i << ", ";
+      arg << "--" << x.long_name() << ']';
+    } else {
+      arg << "--" << x.long_name();
     }
-    if (shorts.size() > 1)
-      for (auto i = 1u; i < shorts.size(); ++i)
-        arg << "-" << shorts[i] << ",";
-    arg << "--" << x.long_name();
-    if (!shorts.empty())
-      arg << "]";
     if (x.has_argument())
       arg << " arg";
     return arg.str();
   };
   // Calculate the max size the argument column
-  std::vector<size_t> arg_sizes;
-  std::transform(
-    long_opts_.begin(), long_opts_.end(), std::back_inserter(arg_sizes),
-    [&](const auto& x) { return build_argument(*x.second).size(); });
-  auto help_column_width = *max_element(arg_sizes.begin(), arg_sizes.end());
+  std::vector<std::string> args;
+  args.reserve(size());
+  for (auto& x: long_opts_)
+    args.emplace_back(build_argument(*x.second));
+  size_t help_column_width = 0;
```

Comment:
The variable name is a bit off, IMO. This values doesn't help the column width, it stores the maximum width. `max_column_width` would be a good fit.